src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -0,0 +1,391 @@
+---
+layout: article
+title: Team invites
+description: Learn how to manage team invites in Appwrite. Implement both client-side email invites and server-side custom flows for team memberships.
+---
+
+Appwrite provides two approaches for adding members to teams: client-side email invites and server-side custom flows. Each approach serves different use cases and offers unique benefits.
+
+# Client-side email invites {% #client-side %}
+
+Client-side email invites provide a user-friendly way to add members to your team. When creating a membership, Appwrite:
+1. Sends an automated email invitation to the user
+2. Creates a pending membership
+3. Activates the membership when the user accepts
+
+Client-side invites are ideal when you want a simple, automated process.
```

Comment:
The important thing to emphasize is this is used when you ant a user to be able to invite someone to have access to something.

Reply:
Got it. Done